packages/drupal/gutenberg_blocks/src/blocks/teaser-list.tsx

```diff
@@ -0,0 +1,114 @@
+import { InnerBlocks, InspectorControls } from 'wordpress__block-editor';
+import { registerBlockType } from 'wordpress__blocks';
+import {
+  PanelBody,
+  SelectControl,
+  TextControl,
+  ToggleControl,
+} from 'wordpress__components';
+
+// @ts-ignore
+const { setPlainTextAttribute } = silverbackGutenbergUtils;
```

Comment:
We should be able to add typing for this in the `declare global` found in `packages/drupal/gutenberg_blocks/src/index.ts`